.github/workflows/build_and_test.yml

```diff
@@ -277,6 +330,24 @@ jobs:
           python -m pip install -r requirements.txt
           python -m pytest --root-path=../../Dist/examples_bin
 
+      - name: Install Coverage Requirements
+        run: python3 -m pip install gcovr
+
+      - name: Process Coverage Files
+        shell: msys2 {0}
+        run: find . -name "*.gcno" -exec gcov -b -l -p -c {} +
+
+      - name: Create Coberture Report
+        run: gcovr -v -g -k -r . --gcov-ignore-parse-errors --exclude-throw-branches --filter Common --filter Pcap --filter Packet --xml -o coverage.xml
```

Comment:
I don't know why but I tried to use $GCOVR_FLAGS variable but it generated empty report.